mmv1/templates/terraform/encoders/vertex_ai_models.go.tmpl

```diff
@@ -0,0 +1,6 @@
+if obj["sourceModel"] != nil {
+    delete(obj, "labels")
+    delete(obj, "metadata")
+}
```

Comment:
Can this be removed? Do we need a custom encoder?

It looks like this might be something created in the past to manage how some fields can only be sent on update and cannot be set on create?

Reply:
after testing without you're right to point out that we don't need this in this PR. I believe this makes sense to include when involving the `:UPLOAD` method. removing this file for this PR only. It can be looked at once more in the `UPLOAD` PR.

[Where it's referenced in UPLOAD PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/12075/files#diff-50cd81d9be3f9dff6ce50d2aafd8bc3ef02dbca3aa9f3615d2b946a7a440a75bR2-R18)